web-app/src/app/routes/monitor/monitor-form/monitor-form.component.html

```diff
@@ -158,6 +158,39 @@
         </nz-form-control>
       </nz-form-item>
 
+      <nz-divider *ngIf="monitor.app === 'prometheus'"></nz-divider>
+      <!-- grafana is enabled -->
+      <nz-form-item *ngIf="monitor.app === 'prometheus'">
+        <nz-form-label nzSpan="7" nzFor="grafana" [nzTooltipTitle]="'monitor.grafana.enabled.tip' | i18n">
+          {{ 'monitor.grafana.enabled.label' | i18n }}
+        </nz-form-label>
+        <nz-form-control nzSpan="8">
+          <nz-switch [(ngModel)]="monitor.grafanaDashboard.enabled" name="grafana" id="grafana"></nz-switch>
+        </nz-form-control>
+      </nz-form-item>
+      <!-- upload grafana dashboard json -->
+      <nz-form-item *ngIf="monitor.grafanaDashboard.enabled && monitor.app === 'prometheus'">
+        <nz-form-label nzSpan="7" nzFor="grafanaJson" [nzTooltipTitle]="'monitor.grafana.upload.tip' | i18n">
+          {{ 'monitor.grafana.upload.label' | i18n }}
+        </nz-form-label>
+        <nz-form-control nzSpan="8">
+          <nz-upload
+            nzName="file"
+            nzListType="text"
+            [nzShowUploadList]="false"
```

Comment:
[nzShowUploadList] It is recommended to change it to true or use other methods to display the files selected by the user